.github/workflows/on-push-lint-charts.yml

```diff
@@ -0,0 +1,67 @@
+name: Lint and Test Charts
+
+on:
```

Comment:
Any reason this is not included in the `pull_request.yml`?

Reply:
I thought it would be simpler to keep the chart publishing as a self-contained workflow, acting only on the master branch. Also, I'm not familiar with `pull_request.yml`, and didn't want to cause confusion by interfering with it. I'm happy to refactor the chart publishing into `pull_request.yml` if you prefer.